python/ql/test/experimental/library-tests/CallGraph/code/class_simple.py

```diff
@@ -25,13 +25,13 @@ def some_classmethod(cls):
 a = A(42)
 
 # calls:A.some_method
-a.some_method()
+a.some_method() # $ pt=A.some_method
 # calls:A.some_staticmethod
-a.some_staticmethod()
+a.some_staticmethod() # $ pt=A.some_staticmethod
 # calls:A.some_classmethod
-a.some_classmethod()
+a.some_classmethod() # $ pt=A.some_classmethod
 
 # calls:A.some_staticmethod
-A.some_staticmethod()
+A.some_staticmethod() # $ pt=A.some_staticmethod
 # calls:A.some_classmethod
-A.some_classmethod()
+A.some_classmethod() # $ pt=A.some_classmethod
```

Comment:
Should the `# calls:`-annotations go?

Also, should there be a bunch of `MISSING: tt`-anotations? That might be a lot of work, though..

Reply:
> Should the # calls:-annotations go?

Yes :+1:

> Also, should there be a bunch of MISSING: tt-anotations? That might be a lot of work, though..

We will get proper annotations when the new call graph is actually implemented, so although this would have been the normal "one commit for tests, one commit for changes" approach, I'll skip over it this time.

Reply:
That is fine.